packages/compiler-vapor/src/generate.ts

```diff
@@ -370,9 +372,11 @@ function genOperation(oper: OperationNode, context: CodegenContext) {
 }
 
 function genSetProp(oper: SetPropIRNode, context: CodegenContext) {
-  const { push, pushWithNewline, vaporHelper } = context
+  const { push, pushWithNewline, vaporHelper, helper } = context
   pushWithNewline(`${vaporHelper('setAttr')}(n${oper.element}, `)
+  if (oper.camel) push(`${helper(helperNameMap[CAMELIZE])}(`)
```

Comment:
It should be a string literal.
```suggestion
  if (oper.camel) push(`${helper('camelize')}(`)
```